commands/qemu-guest-tools/cmd.go

```diff
@@ -94,3 +129,28 @@ func (cmd) Execute(arguments map[string]interface{}) bool {
 
 	return true
 }
+
+func convertSimpleJSONTypes(val interface{}) interface{} {
```

Comment:
There's not a generic way to do something like this? Maybe a comment to explain what's going on here in general.